pkg/live/example-resource-group-crd.yaml

```diff
@@ -18,8 +18,6 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  annotations:
-    controller-gen.kubebuilder.io/version: v0.2.5
```

Comment:
It was good to have this attribution to track which version of controller-gen was used to generate this CRD.

Are we still using controller-gen or some other tooling to generate these ?

Reply:
This is basically documentation of an example of a `ResourceGroup` CRD (using apiextensions.k8.io/v1beta1). I'll re-add the annotation.